### PR TITLE
Add compat error for changed devtools paths, fixes #1165.

### PR DESCRIPTION
**Pull request by EnTeQuAk**

### CODE REVIEW

File: tests/compat/test_gecko44.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from helper import CompatTestCase
+from validator.compat import FX44_DEFINITION
+
+
+class TestFX44Compat(CompatTestCase):
+    """Test that compatibility tests for Gecko 44 are properly executed."""
+
+    VERSION = FX44_DEFINITION
+
+    def test_new_devtools_llocation_gre(self):
```
Comment (magopian) [inline tests/compat/test_gecko44.py]: typo: `location`

File: tests/compat/test_gecko44.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        self.run_script_for_compat("""
+            var devtools = Cu.import("resource://gre/modules/devtools/shared/Loader.jsm", {}).devtools;
+        """)
+        self.assert_silent()
+        self.assert_compat_error()
+
+    def test_new_devtools_llocation_from_root(self):
```
Comment (magopian) [inline tests/compat/test_gecko44.py]: same typo here

**Comment by magopian** [comment]
r+wc

**Comment by EnTeQuAk** [comment]
merged in master

**Pull request closed without merging**